.github/workflows/pythonpackage.yml

```diff
@@ -86,15 +80,9 @@ jobs:
         source /tmp/setup_develop/bin/activate
```

Comment:
Likewise, `setup.py develop` can be replaced with `pip install -e .` You should not need to install dependencies manually first.